website/react-magma-docs/src/pages/api/toggle-button-group.mdx

```diff
@@ -0,0 +1,229 @@
+
+```typescript
+import React from 'react';
+import { ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import { CheckIcon } from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup>
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton
+        defaultChecked
+        aria-label="Check icon"
+        icon={<CheckIcon />}
+      />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## Single Selection
+
+If the prop `singleSelect` is used, then toggles will behave like a radio selection.
+
+```typescript
+import React from 'react';
+import { ButtonSize, ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import { CheckIcon } from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup noSpace singleSelect>
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## Exclusive Selection
+
+When `exclusive` is enabled, it ensures a radio style ToggleButtonGroup has at least one selected state.
+
+```typescript
+import React from 'react';
+import { ButtonSize, ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import {
+  FormatAlignLeftIcon,
+  FormatAlignCenterIcon,
+  FormatAlignRightIcon,
+  FormatAlignJustifyIcon,
+  SettingsIcon,
+} from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup noSpace exclusive singleSelect>
+      <ToggleButton aria-label="Left align" icon={<FormatAlignLeftIcon />} />
+      <ToggleButton
+        aria-label="Center align"
+        icon={<FormatAlignCenterIcon />}
+      />
+      <ToggleButton aria-label="Right align" icon={<FormatAlignRightIcon />} />
+      <ToggleButton
+        aria-label="Justify align"
+        icon={<FormatAlignJustifyIcon />}
+      />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## Size
+
+Three sizes are configurable with the `ButtonSize` prop between `small`, `medium` (default), and `large` across the group.
+
+```typescript
+import React from 'react';
+import {
+  ButtonSize,
+  Container,
+  ToggleButton,
+  ToggleButtonGroup,
+} from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Container style={{ padding: '12px' }}>
+      <ToggleButtonGroup>
+        <ToggleButton size={ButtonSize.small}>Toggle 1</ToggleButton>
+        <ToggleButton size={ButtonSize.medium}>Toggle 2</ToggleButton>
+        <ToggleButton size={ButtonSize.large}>Toggle 3</ToggleButton>
+      </ToggleButtonGroup>
```

Comment:
Because this is the Group doc, I would show the example on the group: 
```suggestion
      <ToggleButtonGroup size={ButtonSize.small}>
        <ToggleButton>Toggle 1</ToggleButton>
        <ToggleButton>Toggle 2</ToggleButton>
        <ToggleButton size={ButtonSize.large}>Toggle 3</ToggleButton>
      </ToggleButtonGroup>
```